### DISCUSSION TITLE
Vertical target area for the Smart clip resize and the speed change is too large

### DISCUSSION THREAD

**Discussion by petersampsonaudacity** [audacity/audacity#5322] [General]
### Bug description
The vertical target area for the Smart clip resize and the speed change is too large (the new square brackets with arrow-heads cursor).

This target area extends way down into the waveform area down to about 0.4 on the vertical scale
a) this is much larger than the horizontal target area for the same click&drag cursor.
b) it is inconsistent with the time-shift gloved-had cursor which only appears when the mouse position is within the vertical bounds of the clip-handle

This led to a recent Forum poster wondering why he could not select within a clip from the clip edge - it turned out that he was selecting too high (above the 0.4 level in the waveform.  This becomes even more of a problem if you collapse tracks or make them very small vertically _(which is often the case in multi-track projects)_.
See: https://forum.audacityteam.org/t/failure-or-only-mystery-with-the-yellow-bar-prompt/85262
The user was confused as to why they weren't consistently getting the yellow snap-guide.

### Steps to reproduce
1.  get some audio mono shows this clearer - but also visible with stereo)
2. make audacity window full screen
3. View > Track Size > Fit to Height
4. Place mouse cursor at T=0 and at the top of the Audacity window 
5. Observe the click&drag square brackets with arrow-heads
6. slowly move the cursor down keeping it at T=0
7. Observe: the cursor only changes to the I-beam Selection cursor when you reach (plus) 0.40

### Expected behavior

A much smaller vertical target area for the click&drag resize/change speed cursor perhaps just the extent of the clip-handle - or perhaps extending only to (plus) 0.90

### Actual behavior
The vertical target size for  the click&drag resize/change speed cursor is unnecessarily large and obtrudes into most working waveforms - thus making selections from the edge of clips unnecessarily difficult/

### Audacity Version
3.4.0 master

### Operating system
Windows 10 - but assume all OS

### Additional context
Flagging: @LWinterberg @Tantacrul  as a UX issue

**Comment by LWinterberg** [comment]
Having it only in the top area is likely going to conflict with fade handles which want to live in those areas. It is probable that a future version will have the top 10% for fade handles, and then the trim hitbox down to the audio center line.

**Comment by Tantacrul** [comment]
I've read the description but I'm not yet convinced I fully understand the issue.

Has there been a change to the 70/30 split we put in place for selecting / trimming or is this a different issue?

I really do think that the faster we build dedicated handles for trimming the better. Certainly out of scope for this release.

**Comment by LWinterberg** [comment]
There has not been a change, it was just a user being confused as to why their cursor would "randomly" change between the Ibeam and the trim cursors and didn't make the connection that it'd be dependent on clip height
